docs/workflow_solutions/using_anaconda.md

```diff
@@ -18,6 +18,26 @@ Anaconda can also install Pip and record which Pip packages are installed, so An
     If using Anaconda on Cheaha, please see our [Anaconda on Cheaha page](../cheaha/software/software.md#anaconda-on-cheaha) for important details and restrictions.
 <!-- markdownlint-enable MD046 -->
 
+## Anaconda best practices for environment reproducibility
+
+Creating, activating and managing environments with Anaconda can be challenging, especially when aiming for reproducibility across different systems, platforms, and projects. To tackle these challenges, below is a list of consolidated best practices drawn from some practical experiences:
+
+1. Using Conda and Pip Wisely: While using Cheaha, or any other platform prioritise using Conda packages as comprehensive compatibility and dependency checks are done during installation. Such that during project implementation, the required libraries function properly and would not throw up compatibility and dependency errors. Use Conda packages instead of Pip when possible as they significantly offer a reduced risk of breaking environments. If Pip packages are necessary for your work, install them after all Conda packages to minimize conflicts and breakage of your environment.
```

Comment:
Include a sentence to state, how packages are installed in order from the yml file.